app/src/main/java/no/usn/mob3000/domain/model/social/FriendData.kt

```diff
@@ -0,0 +1,14 @@
+package no.usn.mob3000.domain.model.social
+
+import kotlinx.datetime.Instant
+
+/**
+ * @author Husseinabdulameer11
+ * created on 05.11.2024
+ **/

```

Comment:
Incorrect creation KDoc property, and date does not follow ISO8601.

Additionally, KDoc should have a short description of what it is.